x-pack/package.json

```diff
@@ -160,6 +160,7 @@
     "d3": "3.5.6",
     "d3-scale": "1.0.6",
     "dataloader": "^1.4.0",
+    "date-fns": "^1.30.1",
```

Comment:
Mainly because

- it's functional
- it's immutable
- it's faster
- we can bundle what we need and not the whole lib 

https://medium.com/@k2u4yt/momentjs-vs-date-fns-6bddc7bfa21e

Reply:
Thanks, I read the link and have a good idea of why date-fns is better than moment. I'd be in support of exploring migrating over. :)

In this particular situation, is it worth adding the extra dependency though? Do we gain anything significant, e.g. noticeable performance improvement? If not, then I think we should just stick with moment (warts and all) to avoid adding another dependency.